packages/clerk-js/package.json

```diff
@@ -76,45 +77,47 @@
     "@types/sass": "^1",
     "@types/testing-library__jest-dom": "^5",
     "@types/testing-library__react-hooks": "^3",
-    "@types/webpack-dev-server": "^3",
-    "@types/webpack-env": "^1.15.1",
-    "autoprefixer": "^10.0.1",
-    "babel-loader": "^8.1.0",
-    "babel-plugin-module-resolver": "^4.0.0",
-    "bundlewatch": "^0.3.2",
-    "css-loader": "^5.0.0",
-    "fork-ts-checker-webpack-plugin": "^5.2.1",
+    "@types/webpack-dev-server": "^4.7.2",
+    "@types/webpack-env": "^1.16.4",
+    "autoprefixer": "^10.4.7",
+    "babel-loader": "^8.2.5",
+    "babel-plugin-module-resolver": "^4.1.0",
+    "bundlewatch": "^0.3.3",
+    "css-loader": "^6.7.1",
+    "esbuild": "^0.14.39",
+    "esbuild-loader": "^2.19.0",
+    "fork-ts-checker-webpack-plugin": "^7.2.11",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^27.4.7",
     "jsonwebtoken": "^8.5.1",
     "node-fetch": "^3.2.0",
-    "postcss": "^8.1.4",
-    "postcss-loader": "^4.0.4",
-    "postcss-prefixer": "^2.1.2",
+    "postcss": "^8.4.13",
+    "postcss-loader": "^6.2.1",
+    "postcss-prefixer": "^2.1.3",
     "querystring-es3": "^0.2.1",
     "react-refresh": "^0.10.0",
-    "sass": "^1.28.0",
-    "sass-loader": "^10.0.5",
+    "sass": "^1.51.0",
+    "sass-loader": "^12.6.0",
     "semver": "^7.3.5",
     "style-loader": "^2.0.0",
     "ts-jest": "^27.1.3",
     "type-fest": "^0.20.2",
     "typescript": "^4.6.2",
-    "webpack": "^5.64.1",
-    "webpack-cli": "^4.9.1",
-    "webpack-dev-server": "^4.5.0",
+    "webpack": "^5.72.1",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.9.0",
     "webpack-merge": "^5.8.0"
   },
   "browserslist": "last 2 versions, ios_saf > 12, Safari > 12, > 1%, not dead, not ie > 0",
   "bundlewatch": {
     "files": [
       {
         "path": "./dist/clerk.js",
-        "maxSize": "148kB"
+        "maxSize": "162kB"
```

Comment:
❓ Does this mean that the new bundle is bigger till we remove hot-loading?

Reply:
The prod bundle is not affected form these changes. Bundlewatch was disabled these values hadn't been updated for a couple of months now.

The bundle size on main is at 159kB, same as this branch